tools/dev-django.sh

```diff
@@ -15,6 +15,7 @@ fi
 
 echo "Installing dependencies"
 apt-get update && apt-get install -y g++ unixodbc-dev # pyodbc build dependencies
+apt-get install -y ldap-utils # ldap dependencies
```

Comment:
`dev-django.sh` is only used for running at development time. For any dependencies that are required in production (which I assume LDAP utils falls under), consider following the same pattern that we have for the MS SQL dependencies:

https://github.com/doccano/doccano/blob/dce6f035a2ea93aedc1ada064e68effdd04420ed/Dockerfile#L46-L47

Reply:
Any progress on this?